djstripe/models.py

```diff
@@ -509,40 +511,45 @@ def sync_current_subscription(self, cu=None):
 
     def update_plan_quantity(self, quantity, charge_immediately=False):
         self.subscribe(
-            plan=djstripe_settings.plan_from_stripe_id(
-                self.stripe_customer.subscription.plan.id
-            ),
+            plan=self.stripe_customer.subscription.plan.id,
             quantity=quantity,
             charge_immediately=charge_immediately
         )
 
-    def subscribe(self, plan, quantity=1, trial_days=None,
+    def subscribe(self, stripe_plan_id, quantity=1, trial_days=None,
```

Comment:
Since people will have to change their code to use this anyway, I think it should require a Plan instance to be passed in. Make sure to add a docstring with param and type descriptors (see Sphinx documentation)